src/Libraries/CoreNodes/List.cs

```diff
@@ -683,6 +684,72 @@ public static IList Transpose(IList lists)
             return transposedList;
         }
 
+        private static bool IsNull(IList list)
+        {
+            return list.Cast<object>().All(el => el == null);
+        }
+
+        /// <summary>
```

Comment:
The XML documentation will display on the node, so here we need to provide more information about what this node does, what are inputs and what is return value.